### DISCUSSION TITLE
Card: Neocolonialism

### DISCUSSION THREAD

**Discussion by RebelusQuo** [RebelusQuo/the-wall-is-down#5] [Ideas]
- Clarification: Is it "up to" to avoid VP redistribution from removing VP below zero?
- No, it's "up to" cause in 2-player games, the West player only loses 2VP.
- Ok, what about this text, which also replaces "may" with "up to":
    > **Only one West player partaking**: West player loses 2 VP.
    > **Otherwise**: West players lose 3 VP, no more than 2 VP per player.
    > Each West player adds up to 1 Influence in any country.

**Comment by rodrigoSantamaria** [comment]
I like that wording, I think you can settle it like that.

**Comment by RebelusQuo** [comment]
Acknowledged. In addition, wherever additional instructions follow an "if/else" formulation the next sentence starts with "Then ..", so here it's
> ..
> Then each West player ..
